importGameToLichess.js

```diff
@@ -15,6 +16,14 @@
     if (!shareButton) {
         shareButton = document.getElementsByClassName("icon-font-chess share live-game-buttons-button")[0];
     }
+    if(!shareButton) {
+        // in case of chess.com/live#g=
+        shareButton = document.getElementsByClassName("icon-font-chess share game-buttons-button")[0];
+    }
+    if(!shareButton) {
```

Comment:
This is technically not correct I am pretty sure, since the fact that we cannot find the button does not necessarily mean that pgn is invalid. Gonna still push it since this way of pulling a pgn will prob change in the future

Reply:
Agreed, "Unable to locate the game PGN" might be a better alert message.
Thank you 👍 🎉